src/utils/change_rundir.cpp

```diff
@@ -39,7 +42,14 @@ void ChangeRunDir(const char *pdir) {
 
   if (pdir == nullptr || *pdir == '\0') return;
 
-  mkdir(pdir, 0775);
+  if (!fs::exists(pdir)) {
+    if (!fs::create_directories(pdir)) {
```

Comment:
This create the directory with permissions 777, doesn't it (see https://en.cppreference.com/w/cpp/filesystem/create_directory "Creates the directory p as if by POSIX [mkdir()](http://pubs.opengroup.org/onlinepubs/9699919799/functions/mkdir.html) with a second argument of static_cast<int>([std::filesystem::perms::all](http://en.cppreference.com/w/cpp/filesystem/perms))"?
That's a little bit too permissive from my point of view and I'd feel more happy with 775 as before.

Reply:
Yes we can do next call fs::permissions to do that. I will do it

Reply:
Lets use a bit more secure 0755 permission. This will allow group users to read and cd into this directory but not write into it. Done in 97d088a4